app/scripts/lib/manifestFlags.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Get the runtime flags that were placed in manifest.json by alterBuiltManifest.ts
+ *
+ * @returns flags if they exist, otherwise an empty object
+ */
+export function getManifestFlags() {
```

Comment:
Should we create a type so we know what flags are available at runtime?

Reply:
Done